docs/user/images.md

```diff
@@ -26,3 +27,13 @@ $ magnum-cluster-api-image-builder --version v1.26.2
 
 In the example above, this command will build the images for Kubernetes version
 `v1.26.2`. The `--version` flag is optional and defaults to `v1.26.2`.
+
+### Building Images (New)
+
+The images used by the Cluster API driver for Magnum are built using the
+[`vexxhost/capo-image-elements`](https://github.com/vexxhost/capo-image-elements)
+project. This project provides a comprehensive and flexible framework for
+constructing Kubernetes-specific images.
+
+You can find pre-built images by this projet at
+https://static.atmosphere.dev/artifacts/magnum-cluster-api/.
```

Comment:
Can you bring this section above and remove the "(New)" and then move the deprecated one underneath it and keep the deprecated comment, so people run into the new thing first. :)